### PR TITLE
[Tests-Only] Acceptance tests to export the local storages mounts to a file

### PR DESCRIPTION
**Pull request by jasson99**
## Description
Add acceptance tests to export the local storages mounts to a file

## Related Issue
#36704 
#37054 

## How Has This Been Tested?
CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -1912,6 +1912,60 @@ public function theAdministratorCreatesFileWithContentInLocalStorageUsingTheTest
...
+			[
+				'file' => "/$path",
+				'content' => $commandOutput
+			],
+			$this->getOcsApiVersion()
+		);
+	}
+
+	/**
+	 * @When the administrator has created a file :path with the last exported content using the testing API
```
Comment (dpakach) [inline tests/acceptance/features/bootstrap/FeatureContext.php]:
```suggestion
	 * @Given the administrator has created a file :path with the last exported content using the testing API
```

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -1912,6 +1912,60 @@ public function theAdministratorCreatesFileWithContentInLocalStorageUsingTheTest
...
+			$user,
+			$this->getAdminPassword(),
+			'POST',
+			"/apps/testing/api/v1/file",
+			[
+				'file' => "/$path",
+				'content' => $commandOutput
+			],
+			$this->getOcsApiVersion()
+		);
```
Comment (dpakach) [inline tests/acceptance/features/bootstrap/FeatureContext.php]: DRY this

File: tests/acceptance/features/cliLocalStorage/importLocalStorage.feature [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@cli @skipOnLDAP @local_storage
+Feature: import exported local storage mounts from the command line
+  As an admin
+  I want to import exported local storage mounts from the command line
+  So that I can create previously exported local storage mounts
+
+  @issue-37054
+  Scenario: export the created mounts to a file
```
Comment (dpakach) [inline tests/acceptance/features/cliLocalStorage/importLocalStorage.feature]:
```suggestion
  Scenario: import local storage mounts from a file
```

File: tests/acceptance/features/cliLocalStorage/exportLocalStorage.feature [hunk 4]
```diff
@@ -4,14 +4,21 @@ Feature: export created local storage mounts from the command line
...
       | /local_storage2    | Local   | None               | datadir:      |                      | All             |                  |
       | /new_local_storage | Local   | None               | datadir:      |                      | All             |                  |
-      | /local_storage     | Local   | None               | datadir:      | enable_sharing: true | All             |                  |
\ No newline at end of file
+      | /local_storage     | Local   | None               | datadir:      | enable_sharing: true | All             |                  |
+
+  Scenario: export the created mounts to a file
+    When the administrator exports the local storage mounts using the occ command
+    And the administrator creates a file "data/exportedMounts.json" with the last exported content using the testing API
+    Then the file "data/exportedMounts.json" with the last exported content should exist in the server root
```
Comment (dpakach) [inline tests/acceptance/features/cliLocalStorage/exportLocalStorage.feature]: This scenario is not really needed IMO. The system is not really responsible for creating the file rather just outputs the content on the standard output and we already have the test for checking the output is correct.

File: tests/acceptance/features/bootstrap/FeatureContext.php (hunk 1 above)
Comment (jasson99) [reply tests/acceptance/features/bootstrap/FeatureContext.php]: fixed

File: tests/acceptance/features/bootstrap/FeatureContext.php (hunk 2 above)
Comment (jasson99) [reply tests/acceptance/features/bootstrap/FeatureContext.php]: fixed

File: tests/acceptance/features/cliLocalStorage/importLocalStorage.feature (hunk 3 above)
Comment (jasson99) [reply tests/acceptance/features/cliLocalStorage/importLocalStorage.feature]: fixed

File: tests/acceptance/features/cliLocalStorage/exportLocalStorage.feature (hunk 4 above)
Comment (jasson99) [reply tests/acceptance/features/cliLocalStorage/exportLocalStorage.feature]: fixed

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by phil-davis — APPROVED** [review]

**Pull request merged**